wordpress-mobile/WordPress-iOS
# [Stats Refresh] Periods loading view

Refs. #10380

This PR adds the loading view in `SiteStatsPeriodTableViewController`
The behaviour is the same implemented [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/11597)

![stats-periods-loading-view](https://user-images.githubusercontent.com/912252/57163778-0aa6f700-6dea-11e9-966e-82d163a5d8a0.gif)

## Error Handling
I will push another PR for the error handling. This because it requires some change on the Store.

## To test:
• From My Sites select a site and open its dashboard
• Select Stats to open the Stats section.
• Select one of the tab period tab _Days, Weeks, Months, Years_.
• You should see the loading view
• The loading view should disappear as soon the data is loaded

Update release notes:
